feat(locale): 🎸 add dateRange pipe

denisyilmaz opened the pull request:
Implementation of the Intl.DateTimeFormat.prototype.formatRange() method

BREAKING CHANGE: 🧨 addition of ES2021.Intl into the typescript configuration

✅ Closes: #763

<!--
Make sure the PR is structured as followed:
[docs/feat/fix/...](package): description
-->

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngneat/transloco/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A

## What is the new behavior?

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->

## Other information

denisyilmaz committed:
feat(locale): 🎸 add dateRange pipe

Implementation of the Intl.DateTimeFormat.prototype.formatRange() method

BREAKING CHANGE: 🧨 addition of ES2021.Intl into the typescript configuration

✅ Closes: #763

denisyilmaz commented:
any plan to get this merged?

shaharkazaz commented:
@denisyilmaz Hey, sorry for not replying I'm currently unavailable. There might be another breaking change introduced in #768 so I think we will release both together as all the other libraries are bumped as well.
I'll try to find the time to review this soon. can you please update from master?

denisyilmaz committed:
Merge branch 'master' into master

denisyilmaz committed:
feat: update transloco-date-range.pipe.ts

- feat: fallback to single date translocoDate pipe if no endDate
- fix: linting
